.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+## What changes were made
+
+-
+
+## How to test or check results
+
+- URL or instructions
+
+## Checks
+- [ ] All content model changes are done through [scripted migrations](../readme.md#scripted-migrations)
+- [ ] I've performed a self-review on my changes
+- [ ] All PR checks have passed
```

Comment:
Think we can leave these two out, as every Voorhoede devs are (or should be) used to this.